web/pdf_viewer.css

```diff
@@ -39,11 +44,49 @@
 
 .pdfViewer.singlePageView {
   display: inline-block;
+  margin: 0;
 }
 
 .pdfViewer.singlePageView .page {
   margin: 0;
   border: none;
+  display: block;
+}
+
+.pdfViewer.scrollHorizontal, .pdfViewer.scrollWrapped {
+  margin: 4px;
+}
+
+.pdfViewer.removePageBorders {
+  margin: 0;
+}
+
+.pdfViewer.scrollWrapped {
```

Comment:
Lines 64 - 66 can be removed if you restrict `white-space: nowrap` to `.pdfViewer.scrollHorizontal` instead of `.pdfViewer`. Since not wrapping is the exceptional case (rather than the default), remove `white-space:nowrap` from line 20 and add `.pdfViewer.scrollHorizontal {white-space:nowrap;}`